ElectronicObserverTypes/Extensions/ShipDataExtensions.cs

```diff
@@ -80,6 +80,8 @@ EquipmentTypes.MainGunLarge or
 	public static bool HasSurfaceRadar(this IShipData ship) => ship.AllSlotInstance
 		.Any(e => e?.MasterEquipment.IsSurfaceRadar is true);
 
+	public static bool HasAirRadar(this IShipData ship) => ship.HasAirRadar(1);
```

Comment:
It's probably better to just change the signature of HasAirRadar to:
public static bool HasAirRadar(this IShipData ship, int count = 1)